public/index.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Composer\Config\Builder;
+use Yiisoft\Di\Container;
+use Yiisoft\Http\Method;
+use Yiisoft\Yii\Web\Application;
+use Yiisoft\Yii\Web\Config\EventConfigurator;
+use Yiisoft\Yii\Web\SapiEmitter;
+use Yiisoft\Yii\Web\ServerRequestFactory;
+
+$autoload = dirname(__dir__) . '/vendor/autoload.php';
+
+if (!is_file($autoload)) {
+    die('You need to set up the project dependencies using Composer');
+}
+
+require_once $autoload;
+
+// Don't do it in production, assembling takes it's time
+Builder::rebuild();
+$startTime = microtime(true);
+$container = new Container(
+    require Builder::path('web'),
+    require Builder::path('providers-web')
```

Comment:
How about merge `web` into `providers-web` by default?

Reply:
you have to ask @yiiliveext

Reply:
> How about merge `web` into `providers-web` by default?

These are different things.

Reply:
Oh, yes definitely. It's my mistake